analyses/cms-open-data-ttbar/analysis.py

```diff
@@ -146,63 +146,44 @@ def define_trijet_mass(df: ROOT.RDataFrame) -> ROOT.RDataFrame:
     )
 
     # Build trijet combinations
-    df = df.Define("Trijet", "ROOT::VecOps::Combinations(Jet_pt[Jet_pt_mask],3)")
-    df = df.Define("nTrijet", "Trijet[0].size()")
+    df = df.Define("Trijet", "Combinations(Jet_pt[Jet_pt_mask],3)")
```

Comment:
since we are touching this line let's add consistent spacing:
```suggestion
    df = df.Define("Trijet", "Combinations(Jet_pt[Jet_pt_mask], 3)")
```